docs/user-guide/complementary-tools.md

```diff
@@ -2,18 +2,15 @@
 
 ## Editor plugins
 
-- [linter-stylelint](https://github.com/AtomLinter/linter-stylelint) - An Atom Linter plugin for stylelint.
-- [SublimeLinter-contrib-stylelint](https://github.com/kungfusheep/SublimeLinter-contrib-stylelint) - A Sublime Text plugin for stylelint.
-- [vscode-stylelint](https://github.com/shinnn/vscode-stylelint) - A Visual Studio Code extension for stylelint.
+- [linter-stylelint](https://github.com/AtomLinter/linter-stylelint) - an Atom Linter plugin for stylelint.
+- [SublimeLinter-contrib-stylelint](https://github.com/kungfusheep/SublimeLinter-contrib-stylelint) - a Sublime Text plugin for stylelint.
+- [vscode-stylelint](https://github.com/shinnn/vscode-stylelint) - a Visual Studio Code extension for stylelint.
 
 ## Runners/loaders
 
-- [gulp-stylelint](https://github.com/olegskl/gulp-stylelint) - A gulp plugin for stylelint.
-- [stylelint-loader](https://github.com/adrianhall/stylelint-loader) - A webpack loader for stylelint.
+- [gulp-stylelint](https://github.com/olegskl/gulp-stylelint) - a gulp plugin for stylelint.
 - [stylelint-webpack-plugin](https://github.com/vieron/stylelint-webpack-plugin) - A webpack plugin for stylelint.
```

Comment:
Isn't there a webpack loader to add? https://www.npmjs.com/package/stylelint-loader